docs/section11/Rule11-13.md

```diff
@@ -0,0 +1,34 @@
+# Service_Water_Heating - Rule 11-13  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** FALSE  
+
+**Rule ID:** 11-13  
+
+**Rule Description:** "Service water-heating energy consumption shall be calculated explicitly based upon the volume of service water heating required and the entering makeup water and the leaving service water-heating temperatures. Entering water temperatures shall be estimated based upon the location. Leaving temperatures shall be based upon the end-use  requirements."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE  
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (e)  
+
+**Evaluation Context:** B_RMD - each building  
+**Data Lookup:**   
+**Function Call:**  
+- NONE  
+
+**Applicability Checks:**  
+- check that there is SWH in the proposed model AND that the building has a 24hr operating schedule
+
+## Applicability Checks:  
+- look at each building in the RMD: `for building in B_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each space in the building: `for space in P_RMD.building...:`
+        - check to see if there is at least one SWH use associated with the space: `if len(space.service_water_heating_uses) > 0:`
+            - the rule is applicable: `is_applicable = true`
```

Comment:
can continue to rule assertion as soon as 1 space is found with ServiceWaterHeatingUse